backend/fixtures/election_2.sql

```diff
@@ -1,6 +1,6 @@
 INSERT INTO elections (id, name, location, number_of_voters, category, number_of_seats, election_date, nomination_date,
                        status, political_groups)
-VALUES (2, 'Municipal Election', 'Heemdamseburg', 100, 'Municipal', 29, '2024-11-30', '2024-11-1',
+VALUES (2, 'Municipal Election', 'Heemdamseburg', 100, 'Municipal', 15, '2024-11-30', '2024-11-1',
```

Comment:
Maybe a comment prevents a future us changing this value back to something greater than 19
```suggestion
-- number_of_seats explicitly set to a value less than 19, to be used in elections with less than 19 seats
VALUES (2, 'Municipal Election', 'Heemdamseburg', 100, 'Municipal', 15, '2024-11-30', '2024-11-1',
```